inference-engine/thirdparty/clDNN/CMakeLists.txt

```diff
@@ -80,11 +80,21 @@ set(CLDNN_BUILD__PROJ_LABEL__clDNN "clDNN")
 
 # ===================================== Include/Link directories =======================================
 
-include_directories(
+if(CLDNN_THREADING MATCHES "TBB")
+  include_directories(
+      ${CLDNN_UTILS__RAPIDJSON_INCDIRS}
+      "${CLDNN__CODEGEN_INCDIR}"
+      "${CLDNN__API_DIR}"
+      "${IE_MAIN_SOURCE_DIR}/src/inference_engine/src/"
```

Comment:
I have the fact that someone include internal IE directories 
We need to expose more capabilities to plugin api in order not to include src directories